sdk/pullrequest.yml

```diff
@@ -24,3 +24,5 @@ extends:
     ServiceDirectory: ${{ parameters.Service }}
     CheckAOTCompat: true
     BuildSnippets: true
+    ExcludePaths:
```

Comment:
@benbp Looks like your PR didn't include this change https://github.com/Azure/azure-sdk-for-net/pull/49511

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-net/pull/49550/files